code/game/gamemodes/revolution/revolution.dm

```diff
@@ -60,8 +60,13 @@
 
 /datum/game_mode/revolution/post_setup()
 	var/list/heads = get_living_heads()
+	var/list/sec = get_living_sec()
+	var/weighted_score = min(max(round(heads.len - ((8 - sec.len) / 3)),1),max_headrevs)
 
-	while(heads.len < head_revolutionaries.len) //das vi danya
+	if(weighted_score < 1)
+		weighted_score = 1

```

Comment:
whoops, this doesn't need to be here anymore. It'll be removed before this is merged